deeplink/build.gradle.kts

```diff
@@ -127,3 +130,23 @@ fun MavenPom.addDependencies() = withXml {
         }
     }
 }
+
+artifactory {
+    setContextUrl("https://oss.jfrog.org/artifactory")
```

Comment:
Shouldn't this be `http://oss.jfrog.org` following [here](https://reflectoring.io/publish-snapshots-with-gradle/)